external/miniz/Kconfig

```diff
@@ -0,0 +1,10 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config LIBMINIZ
```

Comment:
Why do you name it as LIB? It is not in the lib folder and is not provided as a library format.

Reply:
Ok. Should I keep it something like `config MINIZ` or `config MINIZ_COMPRESSION` ?

Reply:
I prefer the second one. But how about `COMPRESSION_MINIZ`? When we name something, we usually name like <higher module>_<real name>.

Reply:
Done. Changed to `COMPRESSION_MINIZ`